packages/design-system/src/components/N8nInfoAccordion/InfoAccordion.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+	<div :class="['accordion', $style.container]" >
+		<div :class="{[$style.header]: true, [$style.expanded]: expanded}" @click="toggle">
```

Comment:
Nitpick: Do we need an array for both classes? 
Can it be something like: `:class=[$style.header,  expanded ? $style.expanded : '']`

Reply:
I prefer objects... feels cleaner than tritiary statements